src/App/Handler/Email/AbstractEmailHandler.php

```diff
@@ -36,9 +42,10 @@
      */
     private $mailFromEmail;
 
-    public function __construct(ValidatorInterface $validator, ToggleManager $manager, ContextFactory $contextFactory, \Swift_Mailer $mailer, string $mailFromEmail = null)
+    public function __construct(ValidatorInterface $validator, ToggleManager $manager, ContextFactory $contextFactory, \Swift_Mailer $mailer, string $mailFromEmail = null, TwigEngine $templating)
```

Comment:
`$templating` should come earlier in the list since `$mailFromEmail` has a default value and might not be passed.